pkg/exchange/coinbase/stream_handlers.go

```diff
@@ -262,8 +298,14 @@ func (s *Stream) handleTickerMessage(msg *TickerMessage) {
 	if !s.checkAndUpdateSequenceNumber(msg.Type, msg.ProductID, msg.Sequence) {
 		return
 	}
-	trade := msg.Trade()
-	s.EmitTradeUpdate(trade)
+	bookTicker := types.BookTicker{
+		Symbol:   toGlobalSymbol(msg.ProductID),
+		Buy:      msg.BestBid,
+		BuySize:  msg.BestBidSize,
+		Sell:     msg.BestAsk,
+		SellSize: msg.BestAskSize,
+	}
+	s.EmitBookTickerUpdate(bookTicker)
```

Comment:
line 295 func comment needs to be updated?

Reply:
Yes, you are right.

Reply:
done.